docs/rules.md

```diff
@@ -827,18 +833,18 @@ Path for CRUD methods on a nested resource type MUST follow valid resource namin
 
 Please refer to [path-for-nested-resource.md](./path-for-nested-resource.md) for details.
 
-### PathForPutOperation
-
-For a PUT operation, If a uri segment has subscription, it needs to have a resource group segment as well.
-
-Please refer to [path-for-put-operation.md](./path-for-put-operation.md) for details.
-
 ### PathForResourceAction
 
 Path for 'post' method on a resource type MUST follow valid resource naming, like '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.MyNameSpace/MyResourceType/{Name}/Action'.
 
 Please refer to [path-for-resource-action.md](./path-for-resource-action.md) for details.
 
+### PathForTrackedResourceTypes
+
+For a operation, If a uri segment has subscription, it needs to have a resource group segment as well.
```

Comment:
Consider rephrasing as - `If the URI segment has a subscription, then the URI needs to have a resource group segment as well.`